schema/query.js

```diff
@@ -160,11 +160,10 @@ const RootQuery = new GraphQLObjectType({
             updated_at,
             ...cred
           } = await Credentials.findById(args.id);
-          cred.schoolId = cred.schoolId.toString();
-          const credHash = web3.utils.sha3(JSON.stringify(cred));
+          
           //data will be true or false, depending on validity of credential
-          const data = await contract.methods.verifyCredential(credHash).call();
-          //so you can see what you get for now
+          const data = await contract.methods.verifyCredential(cred.credHash).call();
+          console.log('data', data)
```

Comment:
We'll want to remove this console.log for production

Reply:
Done.